lib/Buzz/Client/AbstractCurl.php

```diff
@@ -49,22 +49,14 @@ protected static function createCurlHandle()
     /**
      * Populates a response object.
      *
-     * @param resource         $curl     A cURL resource
-     * @param string           $raw      The raw response string
+     * @param string           $header   The raw response header
+     * @param string           $body     The response body
      * @param MessageInterface $response The response object
      */
-    protected static function populateResponse($curl, $raw, MessageInterface $response)
+    protected static function populateResponse($header, $body, MessageInterface $response)
```

Comment:
This is a BC break because you are changing the signature of existing extension points